__tests__/workers/cdc/primary.ts

```diff
@@ -3259,8 +3259,8 @@ describe('user streak change', () => {
     currentStreak: 2,
     totalStreak: 3,
     maxStreak: 4,
-    lastViewAt: new Date().getTime(),
-    updatedAt: new Date().getTime(),
+    lastViewAt: new Date().toISOString() as never,
+    updatedAt: new Date().toISOString() as never,
```

Comment:
Shouldn't it cast to `string` in this case?
Or TS doesn't like it?

Reply:
TS doesn't like it.